phoenix-core/src/it/java/org/apache/phoenix/end2end/HashJoinMoreIT.java

```diff
@@ -43,7 +43,7 @@
 
 import com.google.common.collect.Maps;
 
-public class HashJoinMoreIT extends BaseHBaseManagedTimeIT {
+public class HashJoinMoreIT extends BaseHBaseManagedTimeTableReuseIT {
```

Comment:
This class uses unique name for each test and thus can be derived from BaseHBaseManagedTimeTableReuseIT